pkg/app/configs/prover.go

```diff
@@ -21,6 +21,8 @@ type Config struct {
 // ProverConfig contains only base path to circuits folder
 type ProverConfig struct {
 	CircuitsBasePath string `mapstructure:"circuitsBasePath"`
+	UseRapidsnark    bool   `mapstructure:"useRapidsnark"`
+	RapidsnarkPath   string `mapstructure:"rapidsnarkPath"`
```

Comment:
I would expect to have rapidsnark as executable on the system level. If we need to provide a path to binary, it should be the same with snarkjs.

Reply:
done